Add friends likes

oxelen opened the pull request.

oxelen committed:
Создан пакет storage, перенесена логика хранение Film и User

Commit:
init ТЗ 11

Commit:
init ТЗ 11

Commit:
init ТЗ 11

Commit:
init ТЗ 11

Commit:
init ТЗ 11

Commit:
init ТЗ 11

Commit:
init ТЗ 11

litvitnik commented on src/main/java/ru/yandex/practicum/filmorate/controller/UserController.java:
```diff
@@ -2,162 +2,89 @@
 @Slf4j
 public class UserController {
-    private final Map<Long, User> users = new HashMap<>();
+    private final UserStorage userStorage;
```
Контроллер не должен взаимодействовать с репозиторием. В сервисе  нужно реализовать методы добавления/ обновления/получения) С фильмами аналогично)

litvitnik commented on src/main/java/ru/yandex/practicum/filmorate/controller/ErrorHandler.java:
```diff
@@ -0,0 +1,36 @@
+package ru.yandex.practicum.filmorate.controller;
```
Лучше создать пакет error и поместить туда ErrorHandler и ErrorResponse)

Commit:
ТЗ 11
Убрано взаимодействие контроллеров с репозиториями
Для ошибок создан новый пакет error
